src/Diagnostics/FxCop/CSharp/CSharpFxCopRulesDiagnosticAnalyzers.csproj

```diff
@@ -107,4 +106,4 @@
     <Import Project="..\..\..\..\build\VSL.Imports.Closed.targets" />
     <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
It's not the end of the world, but many Unix utils don't like files that don't end in a line ending.
--
Kevin Pilch-Bisson
    http://twitter.com/Pilchie
________________________________
From: Srivatsn Narayanan <notifications@github.com>
Sent: Monday, February 9, 2015 6:08:46 PM
To: dotnet/roslyn
Cc: Kevin Pilch-Bisson
Subject: Re: [roslyn] Create new projects for System.Runtime.Analyzers (#338)


In src/Diagnostics/FxCop/CSharp/CSharpFxCopRulesDiagnosticAnalyzers.csproj<https://github.com/dotnet/roslyn/pull/338#discussion_r24384180>:

> @@ -107,4 +106,4 @@
>      <Import Project="..\..\..\..\build\VSL.Imports.Closed.targets" />
>      <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
>    </ImportGroup>
> -</Project>
> +</Project>


Ah yes. Fixed them.

-
Reply to this email directly or view it on GitHub<https://github.com/dotnet/roslyn/pull/338/files#r24384180>.